virt/virt-4-13-release-notes.adoc

```diff
@@ -60,7 +63,7 @@ The SVVP Certification applies to:
 +
 All VM templates that are included with {VirtProductName} now use this machine type by default.
 
-** For more information, see xref:../virt/upgrading-virt.adoc#virt-rhel-9_upgrading-virt[{VirtProductName} on {op-system-base} 9]. 
+** For more information, see xref:../virt/upgrading-virt.adoc#virt-rhel-9_upgrading-virt[{VirtProductName} on {op-system-base} 9].
```

Comment:
I do not think a link for further information should be provided as a bullet point. Its context does not align with the previous two bullet points. How about the following suggestion:

> OpenShift Virtualization is now based on Red Hat Enterprise Linux (RHEL) 9. For more information, see [OpenShift Virtualization on RHEL 9](https://apinnick.github.io/previews/CNV-29445-FIPS-RN/virt/upgrading-virt.html#virt-rhel-9_upgrading-virt). The following lists details some major changes from Red Hat Enterprise Linux (RHEL) 9: